### ISSUE TITLE
BUG - sev-2 - All - Appointments: Screen reader users with no appointments trapped in broken UI

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#10327]
## What happened?

A user with screen reader on, who does not have upcoming appointments (like +226), will get stuck in a broken UI upon entering the appointments page:
![Image](https://github.com/user-attachments/assets/d5023f25-7a86-4eeb-ac91-54a95b3ef9fe)

You cannot swipe off of that broken link component. 

During investigation:
* I *was* able to directly tap on other portions of the screen (which is not an allowed workaround per our accessibility/usability standards), and if I opened the 'past' tab, then switched back to the upcoming tab, the UI would be correct/workable.
* If at that point I closed and reopened the appts page, it would be back to broken.
* I also confirmed that non-screenreader users have a working UI immediately upon appointments page load.

All of this is visible in [this video of the issue.](https://drive.google.com/file/d/1b22YxfNGBrjlKLIu_9tCrZ_dzfbzEvR8/view?usp=sharing)

_My guess as to why it's broken is something about the UI not re-rendering appropriately when we're condensing the screen reader heading, which is hopefully a relatively recent change (maybe combined with the backend changes for not calling some APIs if you don't have associated facilities?). But_ 🤷‍♀ 

## Specs:

- Device:
- OS:
- User Account: +226
- Accessibility State: screen reader on

## Steps to Reproduce

See video with the issue, above.

## Desired behavior

Should correctly have the UI of "no appointments" for screen reader users (and, if we're waiting on an API call on that screen, should first show a loading page, THEN the 'no appointments' UI)

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- 2 - High

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by alexandec** [comment]
@TKDickson I'm not able to reproduce the issue with user 226, VoiceOver on. I see the usual "You don't have any appointments" screen and can navigate freely:

https://github.com/user-attachments/assets/2ccad79e-8224-44a9-b6e5-16033f684daa

Same result on Android. Maybe there's another issue that's causing this bug? Anything I might be missing?

**Comment by TKDickson** [comment]
@alexandec on today's develop build, I'm still seeing the issue on my device with +226. Light mode and dark mode (I was briefly hopeful it was maybe related to that).

Are you trying on physical devices, or emulators? My iOS device that I quickly attempted this, this morning, is an iPhone 12 on iOS 18.

**Comment by alexandec** [comment]
@TKDickson interesting, I'm running it on my physical iPhone 12 Pro, iOS 18.1.1, VoiceOver on, latest develop branch, building with XCode to my phone. I tried uninstalling the app completely, then doing another build, then logging in as +226. Still works fine. There must be something different in our setups but I'm not sure what

**Comment by DJUltraTom** [comment]
using todays overnight build i was unable to reproduce on ipad mini, or iphone Xs running 18.0.1

**Comment by TKDickson** [comment]
The device where I can consistently reproduce this is being retired/returned to IT; no one else can reproduce it. Closing as 'unable to reproduce'
